debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -302,15 +302,30 @@ public ReplicationStream startStreaming(Lsn offset, WalPositionLocator walPositi
             LOGGER.debug("starting streaming from LSN '{}'", lsn);
         }
 
-        try {
-            return createReplicationStream(lsn, walPosition);
-        }
-        catch (Exception e) {
-            String message = "Failed to start replication stream at " + lsn;
-            if (e.getMessage().matches(".*replication slot .* is active.*")) {
-                message += "; when setting up multiple connectors for the same database host, please make sure to use a distinct replication slot name for each.";
+        int maxRetries = originalConfig.getInteger(PostgresConnectorConfig.MAX_RETRIES);
+        int tryCount = 0;
+        while (true) {
+            try {
+                return createReplicationStream(lsn, walPosition);
+            }
+            catch (Exception e) {
+                String message = "Failed to start replication stream at " + lsn;
+                if (++tryCount > maxRetries) {
```

Comment:
Do we want to retry all errors? IMHO only the replication slot active message is the whan when we should do the retry.

Reply:
The semantic of the retry configuration do not mention to retry only when it is because of an already active slot:
_slot.max.retries: How many times to retry connecting to a replication slot when an attempt fails_

Then I would say for any Exception, since it handles any case where a retry would be needed

Reply:
ok, makes sense!